Documentation/ceph-common-issues.md

```diff
@@ -824,7 +824,6 @@ They are cases where looking at Kubernetes logs is not enough for diverse reason
 
 So for each daemon, `dataDirHostPath` is used to store logs, if logging is activated.
 Rook will bindmount `dataDirHostPath` for every pod.
-As of Ceph Nautilus 14.2.1, it is possible to enable logging for a particular daemon on the fly.
```

Comment:
Do we want to keep Nautilus until 1.8?

Reply:
This was discussed in the community meeting and #7908 is the tracking issue. Originally the proposal was for 1.7, but this just gives a bit longer for people to upgrade and for downstream flexibility.

Reply:
I know but maybe I wasn't clear. If Nautilus is expected to be removed in 1.8 and this PR will be backported (it has the label) then this line should stay.  Ami I missing something?

Reply:
Oh the minimum version of nautilus is already 14.2.5, right? So this comment didn't seem necessary.

Reply:
Ok right, since Nautilus is already the default it's fine removing it then.